OracleTuxedo/salt/samples/bankapp/README.md

```diff
@@ -0,0 +1,54 @@
+
+# Oracle Tuxedo `bankapp` Sample Container Image
+
+This sample extends the Oracle Tuxedo image by creating a "bankapp" application.
+
+## Prerequisites
+
+This image uses the Orace Tuxedo container image `oracle/tuxedo:latest` as its base. Please follow the [Oracle Tuxedo image](https://github.com/oracle/docker-images/tree/main/OracleTuxedo/core) documentation to build the container image before performing the operations described in this document.
+
+## How to Build the Image
+
+Download the parent directory (bankapp) of this README.md file from GitHub to a location of your choice. This location is referenced as `YOUR_APP_DIR` in this document.
+
+```shell
+export TUXDIR=<Tuxedo installation directory when the above mentioned `oracle/tuxedo:latest` image was built>
```

Comment:
```suggestion
export REPO_BASE="/path/to/oracle/docker-images"
export TUXDIR="$REPO_BASE/OracleTuxedo/core/dockerfiles/22.1.0.0.0"
```

You could probably remove the first line if you state that `$REPO_BASE` is the path to where this Git repo has been cloned.

Reply:
TUXDIR here is not from the Tuxedo core docker image. Its from the manually downloaded Tuxedo binaries from the Oracle downloads website and which was installed prior to building the Tuxedo core docker image "oracle/tuxedo:latest"

Reply:
It doesn't really matter where TUXDIR is pointing. I want you to replace `<Tuxedeo installation directory yadda yadda...>` with the actual path you use in the examples in the `oracle/docker-images` repo when building this image.

Reply:
I was able to removed this entire section actually. Moved it to the start.sh where the build happens automatically and hence the user will no longer be required to copy and build the bankapp manually.